mobile/pages/Search.tsx

```diff
@@ -0,0 +1,286 @@
+import React, { useState, useEffect } from 'react';
+import { vw, vh } from 'react-native-expo-viewport-units';
+import { useFonts, Poppins_500Medium } from '@expo-google-fonts/poppins';
+import {  Text, View, TouchableOpacity, StyleSheet, Image, Button } from 'react-native';
+import {Picker} from '@react-native-picker/picker';
+
+
+
+export default function Search( props: any )
+{
+
+
+
+
+useFonts({
+    Poppins_500Medium,
+});
+
+const [state, setState] = useState();
+const [city, setCity] = useState();
+const [resEstados=[], setResEstados] = useState()
+const [markers=[], setMarkers] = useState(false);
+
+
+useEffect(() => {
+    function goToMap() {
+        props.navigation.navigate('Map', {
+            body: markers
+        
+        })
+    }
+
+    if(markers != false)
```

Comment:
i like this pattern for keys, how about adding a lint for mobile? ESlint is not very difficult to configure and does automatic fixes in vscode
```ts
if(condidion){

}
```